src/peer_review/related_models/peer_review_invite_model.py

```diff
@@ -0,0 +1,47 @@
+from django.db import models
+from utils.models import DefaultModel
+from peer_review.models import PeerReviewRequest
+
+
+class PeerReviewInvite(DefaultModel):
+    INVITED = 'INVITED'
+    ACCEPTED = 'ACCEPTED'
+    DECLINED = 'DECLINED'
+
+    REQUEST_STATUS_CHOICES = [
+        (INVITED, INVITED),
+        (ACCEPTED, ACCEPTED),
+        (DECLINED, DECLINED)
+    ]
+
+    invited_user = models.ForeignKey(
```

Comment:
should we also add an email field here for users not on the platform?

Reply:
Ah, good point @lightninglu10

Reply:
Added